Arrowgene.Ddon.Shared/Model/Enemy.cs

```diff
@@ -31,8 +31,10 @@ public class Enemy
         public bool IsAreaBoss { get; set; }
         public uint BloodOrbs { get; set; }
         public uint HighOrbs { get; set; }
+        public string SpawnTime { get; set; }
```

Comment:
Instead of storing the string, the converted numeric values should be the ones being stored